src/components/SettingsMenu.css

```diff
@@ -0,0 +1,104 @@
+.HeaderImage {
```

Comment:
This css is actually just copy pasta from Gridlayout.css. No extra css added. Always a risk to replace big chunks of css because there is no way to know for sure it is still working